documentation/topics/glossary.md

```diff
@@ -0,0 +1,119 @@
+# Glossary
+
+(This is a work in progress and will be expanded over time!)
+
+## Action
+
+An action describes an operation that can be performed for a given resource; it is the _verb_ to a resource's _noun_. Examples of actions:
+
+- User._create_
+- Comment._delete_
+- BlogPost._publish_
+- Article._search_
+
+Ash supports four different types of actions - `create`, `read`, `update` and `destroy` (collectively often abbreviated as CRUD). A resource can define multiple actions per action type, eg. a _publish_ action would be considered an `update` because it is updating an existing instance of a resource.
```

Comment:
I might add something like this: "Actions are much more flexible than simple CRUD, but these four action types serve as templates for anything you might want to do." ?